security-tests/package.json

```diff
@@ -10,7 +10,7 @@
 	"author": "",
 	"license": "ISC",
 	"dependencies": {
-		"@aws-sdk/client-iam": "3.529.1",
+		"@aws-sdk/client-iam": "3.621.0",
```

Comment:
This change seems to be causing unit tests to fail. We should investigate before we merge this change.

Reply:
@its-aazizi pointed out that forked repos don't have access to the github secrets, which makes sense. So I'll close this PR and use the workflow he mentioned.